src/cart/slice/slice.ts

```diff
@@ -1,8 +1,12 @@
 import { createSelector, createSlice, PayloadAction } from "@reduxjs/toolkit";
 import { CartProduct } from "../../products/types";
 import { RootState } from "../../_shared/store";
+import { selectIProductsSlice } from "../../products";
 
-const initialState: CartProduct[] = [{ id: 1, qty: 10 }];
+const initialState: CartProduct[] = [
+  { id: 1, qty: 10 },
+  { id: 2, qty: 10 },
+];
```

Comment:
considering that we will access it by id, we can improve the state structure using a map instead of an array